tests/testerina-integration-test/src/test/resources/command-outputs/unix/DataProviderTest-testArrayDataProviderWithFail.txt

```diff
@@ -6,6 +6,15 @@ WARNING [tests/new-data-provider-tests.bal:(153:9,153:21)] unused variable 'a'
 Running Tests with Coverage
 
 	dataproviders
+
+intArrayDataProviderTest:0 has failed.
```

Comment:
Can we indent this to align with the rest of the output? Ideally it must align with the line dataproviders